.github/workflows/ci.yml

```diff
@@ -31,20 +46,36 @@ jobs:
       - run: python -m pip install -r requirements.txt
       - run: ${{ matrix.lint-command }}
 
-  dist:
+  readme:
```

Comment:
I don't understand this action very well, can you explain it?

Reply:
It creates the source and wheel distributions, to check that the process is well. It also runs readme-renderer, which in ensure that the readme will be correctly rendered on PyPi.